pkg/services/pluginsintegration/clientmiddleware/tracing_middleware_test.go

```diff
@@ -331,6 +334,16 @@ func TestTracingMiddlewareAttributes(t *testing.T) {
 	}
 }
 
+func spanAttributesContains(attribs []attribute.KeyValue, attrib attribute.KeyValue) bool {
```

Comment:
Nit: This could actually be quite handy for other tests as well. Maybe we can move it to `pkg/infra/tracing/test_helper.go` instead?

Reply:
I did consider it, but concluded its better to move/change this when another test needs this.